### ISSUE TITLE
Anytone 878UV is not detected by qdmr

### ISSUE THREAD

**Issue by DaniAngel** [hmatuschek/qdmr#417]
I just installed qdmr on linux mint 20.3.  When I connect the radio via the usb cable adapter, I cannot get qdmr to detect it.  When I click detect, it gives me an error that no matching device was found.  I checked my groups permission with the groups command in the console and dialout is listed.  Is there something else I should try?

**Comment by galagithub** [comment]
can you post a "lsusb" output with the cable inserted and with the cable removed from the PC (complete disconnect).

**Comment by galagithub** [comment]
if the cable style is audio jack like, then that is NOT a passive cable, but a USB to serial converter.

when taking the "lsusb" output with no cable attached, make sure the cable is completely removed/unplugged from the PC.

**Comment by hmatuschek** [comment]
I do not believe, that is a direct issue with qdmr. Move that to the discussions.
